.github/workflows/build.yml

```diff
@@ -13,23 +13,30 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.experimental }}
         name: "PHP ${{ matrix.php }}${{ matrix.symfony != '' && format(', Symfony {0}', matrix.symfony) || '' }}${{ matrix.currency-bundle != '' && format(', CurrencyBundle {0}', matrix.currency-bundle) || '' }}${{ matrix.postgres != '' && format(', PostgresSQL {0}', matrix.postgres) || '' }}"
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
+                php: ["8.1", "8.2", "8.3"]
```

Comment:
Since we're allowing installation with PHP 8.0, I think we should leave at least a single build for it.

Reply:
I would be for upgrading minimum PHP to 8.1 🤔

Reply:
done, I bump php version on Bundle and composer.json